### PR TITLE
Update totp-register.vm

### PR DESCRIPTION
**Pull request by ArsHaider**
Updating page to include information about other authenticators.

## Changes proposed in this pull request:
-Updating page to include information about other authenticators.
-
-

## security considerations
None, just listing the same authenticators that are already listed on our site.

### CODE REVIEW

File: totpauth-impl/src/main/resources/views/totp-register.vm [hunk 1]
```diff
@@ -58,6 +59,13 @@ p.error {
             </div>
           </form>
           <li><a href="https://support.google.com/accounts/answer/1066447" target="_blank">Google Authenticator installation instructions</a></li>
+<li><a href="https://support.authy.com/hc/en-us/articles/115001945848-Installing-Authy-apps" target="_blank">Authy Authenticator installation instructions</a></li>
```
Comment (markdboyd) [inline totpauth-impl/src/main/resources/views/totp-register.vm:62]: Would this be a better link?

https://support.authy.com/hc/en-us/articles/360006303934-Add-a-New-Two-Factor-Authentication-2FA-Account-Token-in-the-Authy-App

The current link just talks about how to install Authy

File: totpauth-impl/src/main/resources/views/totp-register.vm [hunk 2]
```diff
@@ -38,7 +38,8 @@ p.error {
           #end
           <form action="$flowExecutionUrl" method="post">
             #parse("csrf/csrf.vm")
-            <li> Download the Google Authenticator app to your mobile device </li>
+            <li> Download an authenticator application or service to your mobile device that is compatible with RFC 6238. Any application or service which is compatible with RFC 6238 will work. Listed here are some cross device authenticators that are compatible with cloud.gov: Microsoft Authenticator, LastPass, Password Manager, 1password, Authy.
```
Comment (markdboyd) [inline totpauth-impl/src/main/resources/views/totp-register.vm:41]:
```suggestion
            <li> Download an authenticator application or service to your mobile device that is compatible with RFC 6238. Any application or service which is compatible with RFC 6238 will work. Listed here are some cross device authenticators that are compatible with cloud.gov: Microsoft Authenticator, LastPass, 1password, Authy.
```

Password Manager was listed but not described below. I also don't know if that's a real thing?

File: totpauth-impl/src/main/resources/views/totp-register.vm (hunk 1 above)
Comment (ArsHaider) [reply totpauth-impl/src/main/resources/views/totp-register.vm:62]: I agree

Comment (ArsHaider) [reply totpauth-impl/src/main/resources/views/totp-register.vm]: Change made

File: totpauth-impl/src/main/resources/views/totp-register.vm [hunk 3]
```diff
@@ -58,6 +59,13 @@ p.error {
             </div>
           </form>
           <li><a href="https://support.google.com/accounts/answer/1066447" target="_blank">Google Authenticator installation instructions</a></li>
+<li><a href="https://support.authy.com/hc/en-us/articles/360006303934-Add-a-New-Two-Factor-Authentication-2FA-Account-Token-in-the-Authy-App" target="_blank">Authy Authenticator installation instructions</a></li>
+<li><a href="https://support.microsoft.com/en-us/account-billing/set-up-an-authenticator-app-as-a-two-step-verification-method-2db39828-15e1-4614-b825-6e2b524e7c95" target="_blank">Microsoft Authenticator installation instructions</a></li>
+<li><a href="https://support.lastpass.com/s/document-item?language=en_US&bundleId=lastpass&topicId=LastPass/enable-mfa.html&_LANG=enus" target="_blank">LastPass Authenticator installation instructions</a></li>
```
Comment (markdboyd) [inline totpauth-impl/src/main/resources/views/totp-register.vm:64]: Reading this link, it refers to the process of adding a 2nd factor for authentication to your LastPass account. It's not about using LastPass to manage 2-factor authentication to other accounts. Is that definitely supported by LastPass?

Comment (markdboyd) [reply totpauth-impl/src/main/resources/views/totp-register.vm:64]: I think this is the more relevant link: https://support.lastpass.com/s/document-item?language=en_US&bundleId=lastpass&topicId=LastPass/create-totp-vault.html&_LANG=enus

Comment (ArsHaider) [reply totpauth-impl/src/main/resources/views/totp-register.vm]: I agree, change made.

**Review by markdboyd — APPROVED** [review]

**Pull request merged**